### ISSUE TITLE
[Accordion] Background color sl-accordion

### ISSUE THREAD

**Issue by DanielleRameau** [sl-design-system/components#1111]
**Describe the bug**
In dark mode, we apply a transparent hover color. If the accordion is placed on a light background (for example on the documentation website) the color contrast is too low. 

**To Reproduce**
1. Go to 'sanomalearning.design' (in dark mode)
2. Click on 'accordion'
3. Hover the accordion

**Expected behavior**
Apply token: body.background on sl-accordion so it has a stable background-color on all backgrounds.

**Screenshots**
How it looks now:
<img width="764" alt="Screenshot 2024-03-29 at 15 18 40" src="https://github.com/sl-design-system/components/assets/111998351/e25ea92c-382e-4131-a57d-4e449f305fbb">

How it should be:
<img width="733" alt="Screenshot 2024-03-29 at 15 18 18" src="https://github.com/sl-design-system/components/assets/111998351/55424ae4-ea7d-4227-b6a2-4bbe62961e8a">
